packages/convenient_test_manager_dart/lib/misc/setup.dart

```diff
@@ -25,16 +26,22 @@ Future<void> setup({
   bool registerFsService = true,
   bool registerHighlightStoreBase = true,
   bool registerVideoPlayerStoreBase = true,
+  bool registerVmServiceWrapper = true,
+  bool parseConfigFile = true,
 }) async {
-  GlobalConfigStore.config = await GlobalConfigNullable.parse(args: args, headlessMode: headlessMode);
+  if (parseConfigFile) {
+    GlobalConfigStore.config = await GlobalConfigNullable.parse(args: args, headlessMode: headlessMode);
+  } else {
+    GlobalConfigStore.config = GlobalConfigNullable().toConfig();
+  }
```

Comment:
uber-nit

```suggestion
    GlobalConfigStore.config = parseConfigFile 
    ?     await GlobalConfigNullable.parse(args: args, headlessMode: headlessMode)
  : GlobalConfigNullable().toConfig();
```

to avoid two assignments which may be slightly more error prone